src/main/java/org/quantumbadger/redreader/http/okhttp/OKHTTPBackend.java

```diff
@@ -31,11 +35,26 @@
 public class OKHTTPBackend implements HTTPBackend {
 
 	private final OkHttpClient mClient;
+	private static HTTPBackend httpBackend;
+	public static boolean reset = false;
+
+	public static HTTPBackend getHttpBackend(boolean force) {
```

Comment:
Also it might be better to split this into `getHttpBackend()` and `recreateHttpBackend()` to get rid of the parameter, but that's just nitpicking :)